src/Bundle/Command/DebugGridCommand.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\GridBundle\Command;
+
+use Sylius\Component\Grid\Provider\GridProviderInterface;
+use Symfony\Component\Console\Attribute\AsCommand;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Helper\Dumper;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\PropertyAccess\PropertyAccess;
+
+#[AsCommand(name: 'sylius:debug:grid')]
+final class DebugGridCommand extends Command
+{
+    public function __construct(
+        private readonly GridProviderInterface $gridProvider,
+    ) {
+        parent::__construct();
+    }
+
+    protected function configure(): void
+    {
+        $this
+            ->addArgument(name: 'grid', mode: InputArgument::REQUIRED, description: 'The grid name of FQCN')
```

Comment:
```suggestion
            ->addArgument(name: 'grid', mode: InputArgument::REQUIRED, description: 'The name or fully-qualified class name (FQCN) of the grid to debug.')
```

<img width="765" height="167" alt="image" src="https://github.com/user-attachments/assets/966ab223-fc2c-42ba-b5fc-edaeab24db67" />